isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc

```diff
@@ -197,44 +184,20 @@ Innerhalb des Service-Gateways existiert keine Geschäftslogik, d.h. die Abbildu
-Wird in so einem Fall ein neuer Service eingeführt, während der alte Service noch verfügbar bleiben muss, müssen die inkompatiblen Verarbeitungslogiken im Anwendungskern parallel erhalten bleiben.
+Wird in so einem Fall ein neuer Service eingeführt, während der alte Service noch verfügbar bleiben muss, müssen die inkompatiblen Verarbeitungslogiken im Anwendungskern parallel unterstützt werden.
 Auch hier enthält das Service-Gateway keine Geschäftslogik.
 
-=== Umsetzung
-Die Java-Klassen und -Interfaces eines Services existieren in allen Versionen der Service-Schnittstelle und unterscheiden sich inhaltlich
-durch die in der neuen Version durchgeführten Änderungen.
-
-NOTE: Für die Versionierung von Schnittstellen gelten gesonderte Vorgaben, die in  xref:werkzeuge:versionierungskontrolle/master.adoc[IsyFact Versionierung] definiert sind.
-
-Zur Veröffentlichung von API-kompatiblen Änderungen wird im Maven `pom.xml` eine einstellige Versionsnummer (Minor) gesetzt.
-Kompatible Änderungen sind beispielsweise Bugfixes, neue Operationen in der Schnittstelle oder neue, optionale Attribute im Datenmodell.
-
-.Realisierung der Versionierungsvorgaben für Schnittstellen bei HTTP Invoker
-[id="listing-versioning-pom", reftext="reftext="{listing-caption} {counter:listings }"]
-[source,xml]
-----
-<dependencies>
-    ...
-    <dependency>
-        <groupId>${Organisation.Domäne.Anwendungsname}</groupId>
-        <artifactId>${Anwendungsname}-${Schnittstellentechnologie}-sst-${Servicename}-v${Major-Version}</artifactId>
-        <version>${Minor-Version}</version>
-    </dependency>
-    ...
-</dependencies>
-----
-
-Bei inkompatiblen Änderungen der Schnittstelle wird die zweistellige Versionsnummer angepasst (Major und Minor); diese wird sowohl in der Artefakt-ID als auch in den Paketnamen der Schnittstelle verwendet.
-Inkompatible Änderungen der Schnittstelle sind z. B. das Entfernen von Attributen oder Operationen oder das Hinzufügen von Pflichtfeldern.
-
-Bei der Implementierung ist zu beachten, dass die Versionsnummer aus dem Package-Namen auch in die Implementierung übernommen wird.
-
 [[grenzen]]
 === Grenzen
-
 Eine Versionierung ist nur dann sinnvoll, wenn kleine Änderungen an der Schnittstelle zwischen den Versionen auftreten.
 Für den Fall, dass sich die Schnittstelle sowohl syntaktisch als auch semantisch grundlegend ändert, sollte anstatt einer neuen Version besser eine eigenständige, neue Schnittstelle entstehen.
 
+Wenn man sich für eine Service-Versionierung entscheidet, so sollte man sich bewusst sein, wie sich diese auf Klassen und Packages abbildet:
+
+* In der Service-Fassade sollten verschiedene Service-Versionen anhand einer zweistelligen Versionsnummer Major.Minor unterscheidbar sein. 
+Bei kompatiblen Änderungen wird die nur Minor Version, bei inkompatiblen Änderungen immer die Major und die Minor Version angepasst.
+
+* Im Anwendungskern dagegen findet man lediglich zu jeder Major-Version eine jeweils neue Service-Implementierung.
```

Comment:
Das haben wir so nicht besprochen und dazu kenne ich auch keine Vorgaben, daher würde ich diesen Satz (Zeile 200) löschen.

Reply:
197 - 200 wäre eine logische Folge dessen 
* was in Abschnitt 6.1 steht, 
* wenn es mit den in 227-228 formulierten Aussagen in Verbindung bringt. 
 die gelöscht wurden.